package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@buildium/angular-elements",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
This probably deserve more than a patch, may be Minor?

Reply:
Makes sense.